sources/api/schnauzer/tests/v2_parse_fail_templates.rs

```diff
@@ -0,0 +1,51 @@
+use schnauzer::v2::{template, Template};
+
+macro_rules! assert_parse_error {
+    ($error:ident, $template:expr) => {
+        match include_str!($template).parse::<Template>() {
+            Err(template::error::Error::$error { .. }) => (),
+            _ => panic!("Did not encounter expected error while parsing template."),
+        };
+    };
+}
+
+#[test]
+fn fails_00_invalid_toml() {
```

Comment:
I appreciate that you have written these as separate tests instead of cycling through directory templates. Separate tests are much better!